src/extractor/extraction_containers.cpp

```diff
@@ -433,9 +439,10 @@ void ExtractionContainers::PrepareEdges(ScriptingEnvironment &scripting_environm
     // Sort edges by start.
     std::cout << "[extractor] Sorting edges by renumbered start ... " << std::flush;
     TIMER_START(sort_edges_by_renumbered_start);
+    std::mutex name_data_mutex;
     stxxl::sort(all_edges_list.begin(),
                 all_edges_list.end(),
-                CmpEdgeByInternalSourceTargetAndName{name_char_data, name_offsets},
+                CmpEdgeByInternalSourceTargetAndName{name_data_mutex, name_char_data, name_offsets},
```

Comment:
So `stxxl::sort` is parallelized? Interesting, didn't know that. Might be worth a comment, since on the calling code there is no indication.